src/App.tsx

```diff
@@ -1,26 +1,228 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState, useCallback } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import * as postService from './api/todos';
+import { Errors } from './components/errors/errors';
+import { Footer } from './components/footer/footer';
+import { ToDoList } from './components/todoList/todoList';
+import { Header } from './components/header/header';
+import { Todo } from './types/Todo';
+import { TodoStatus } from './types/TodoStatus';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
+  const [todosFromServer, setTodosFromServer] = useState<Todo[]>([]);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [status, setStatus] = useState('');
```

Comment:
```suggestion
  const [filter, setStatus] = useState('');
```

Reply:
but if i use 'status' it`s look more logical, no?